aten/src/ATen/native/cpu/IndexKernel.cpp

```diff
@@ -525,6 +524,39 @@ void cpu_hflip_vec(at::TensorIterator& iter) {
   iter.cast_outputs();
 }
 
+template <typename scalar_t>
+void cpu_vflip_memcpy(at::TensorIterator& iter) {
+
+  auto loop2d = [&](char** base, const int64_t *strides, int64_t size0, int64_t size1) {
+
+    static constexpr int ntensors = 3;
```

Comment:
I'm sorry, but where is this 3 coming from? It's confusing me so much.

Reply:
It's the number of input and output tensors: 1 output + 1 input + 1 restrided input: https://github.com/pytorch/pytorch/blob/0f7dca17332152fdd28270eb95398efbe8212ca2/aten/src/ATen/native/TensorTransformations.cpp#L64-L66